src/runtime/id.ts

```diff
@@ -36,6 +36,11 @@ export class IdGenerator {
   static createWithSessionIdForTesting(sessionId: string) {
     return new IdGenerator(sessionId);
   }
+  
+  /** Creates a new ArcId. */
```

Comment:
Don't comment if you're just re-phrasing the name of the method and there's nothing worthwhile to add. Let me know if you disagree here.

Reply:
Sounds good to me